### DISCUSSION TITLE
4.0 diff

### DISCUSSION THREAD

**Discussion by muchgitsuchwow** [fosskers/aura#828] [General]
With aura 3. a PKGBUILD diff was shown and one was asked whether one want to continue. With 4. one is asked whether one wants to see the diff and aura continues right away.

Can we have the version 3 behavior with 4.? By the way I'd remove the `diff` config option. I cannot see any reason why someone doesn't want the PKGBUILD diff to be shown. Anyoo.

**Comment by fosskers** [comment]
I will have to confirm the current behaviour of the diff functionality in 4. I will opt to leave it opt-in though, since not everyone wants to see it. Sometimes it can get quite spammy.
